engine/apps/alerts/models/channel_filter.py

```diff
@@ -126,29 +132,30 @@ def select_filter(cls, alert_receive_channel, raw_request_data, force_route_id=N
                 )
                 pass
 
-        filters = cls.objects.filter(alert_receive_channel=alert_receive_channel)
+        channel_filters = cls.objects.filter(alert_receive_channel=alert_receive_channel)
 
-        satisfied_filter = None
-        for _filter in filters:
-            if satisfied_filter is None and _filter.is_satisfying(raw_request_data):
-                satisfied_filter = _filter
```

Comment:
I didn't see a reason to continue looping over `filters` if `satisifed_filter` is already set and being returned just below